test/transactions_test.rb

```diff
@@ -114,6 +114,18 @@ def test_raise_command_errors_in_multi_exec
     assert_equal "s1", r.get("foo")
   end
 
+  def test_empty_multi_exec
+    result = nil
+
+    redis_mock(:exec => lambda { |*_| "-ERROR" }) do |redis|
+      result =
+        redis.multi do |m|
```

Comment:
np: the formatting here is very weird. A one-liner would be better IMO `result = redis.multi {}`